client/app/iplb/frontends/iplb-frontends-edit.controller.js

```diff
@@ -192,7 +192,9 @@ class IpLoadBalancerFrontendsEditCtrl {
                 break;
             default: break;
         }
-        frontend.port = parseInt(frontend.port, 10);
+        //port can not be integer because it can be simple(80), range (1000-2000) or multi (80,443)
+        //converting it to integer will cause problem if it is range or multi
+        //frontend.port = parseInt(frontend.port, 10);
```

Comment:
HI! Avoid commented code here.
In addition, if you remove the commented code... you can remove the comment as well ;)

Reply:
+1 useless comment since the documented line isn't needed anymore (frontend.port = parseInt(frontend.port, 10);)

Just flush all of it.

Reply:
However I think we might need to check with a regex that the port number is something like 13124 or 234, 123123, 123123 or 234234,234234,234234

Reply:
removed commented code

Reply:
regarding regex validation for port
It's very good idea. I thought about it and spent some time. Port can be between 1 to 49151. And it can also be a range (80-100) or multi (80,443). Regex becomes very complex to support this validation. Since backend API is doing proper validation. I did not add any validation on UI side.